src/NzbDrone.Core/Localization/Core/en.json

```
@@ -538,6 +538,7 @@
   "Ignored": "Ignored",
   "IgnoredAddresses": "Ignored Addresses",
   "Images": "Images",
+  "ImdbId": "IMDB ID",
```

```suggestion
  "ImdbId": "IMDb ID",
```

to follow their branding